clouds/azure/session_clients.py

```diff
@@ -0,0 +1,29 @@
+from azure.identity import ClientSecretCredential
```

Comment:
Please add tests.

Reply:
We want to test that `ClientSecretCredential` works properly given credential values (client id/secret, tenant_id) ?
Not sure what we want to test